util/customTransformer-v1.js

```diff
@@ -35,22 +36,17 @@ async function transform(isolatevm, events) {
     }
   });
 
-  let result;
-  try {
-    const timeoutPromise = new Promise(resolve => {
-      const wait = setTimeout(() => {
-        clearTimeout(wait);
-        resolve("Timedout");
-      }, userTransformTimeout);
-    });
-    result = await Promise.race([executionPromise, timeoutPromise]);
-    if (result === "Timedout") {
-      throw new Error("Timed out");
-    }
-  } catch (error) {
-    throw error;
-  }
-  return result;
+  let setTimeoutHandle;
+  const timeoutPromise = new Promise((_, reject) => {
+    setTimeoutHandle = setTimeout(() => {
+      reject(new Error("Timed out"));
+    }, userTransformTimeout);
+  });
+  return Promise.race([executionPromise, timeoutPromise])
+    .catch(e => {
```

Comment:
this is catch required?

Reply:
It shows `unhandledpromiserejectionwarning` warning during unit tests. added for readibility